ocean_lib/web3_internal/test/test_contract_base.py

```diff
@@ -64,16 +64,22 @@ def test_main(network, alice_wallet, alice_address, dtfactory_address, alice_oce
     assert factory.address == dtfactory_address
     assert factory.events
     assert str(factory) == f"{factory.contract_name} @ {factory.address}"
-    assert "createToken" in factory.function_names
+    assert (
+        "createToken" in factory.function_names
+    ), "The function createToken from the contract does not exist."
     assert "getCurrentTokenCount" in factory.function_names
     assert "getTokenTemplate" in factory.function_names
     assert not factory.is_tx_successful("nohash")
     with pytest.raises(ValueError):
-        assert factory.get_event_signature("noevent")
+        assert factory.get_event_signature(
```

Comment:
please remove the `assert` keyword here. pytest.raises "is" the assert. i.e. asserts that the content will raise the exception. The `factory.get_event...` part should be enough to throw the exception.

Reply:
Missed it, sorry, refactored now.